src/scripts/clipperUI/regionSelector.tsx

```diff
@@ -41,26 +51,29 @@ class RegionSelectorClass extends ComponentBase<RegionSelectorState, ClipperStat
 		this.resetState();
 
 		window.addEventListener("resize", this.resizeHandler);
+		window.addEventListener("mousemove", this.mouseMovementHandler);
+		window.addEventListener("mouseover", this.mouseOverHandler);
 	}
 
 	private onunload() {
 		window.removeEventListener("resize", this.resizeHandler);
+		window.removeEventListener("mousemove", this.mouseMovementHandler);
```

Comment:
I am removing it in the handler itself after we detect it once. But it might make sense to remove it in onunload as well just in case the mouse handler never came over the window.